org.eventb.emf.persistence/src/org/eventb/emf/persistence/SaveResourcesCommand.java

```diff
@@ -104,6 +110,7 @@ protected IStatus doExecute(IProgressMonitor monitor, IAdaptable info) {
 		for (final Resource resource : modifiedResources) {
 			try{
 				resource.save(Collections.emptyMap());
+				resource.setModified(false);
```

Comment:
This need to be investigated further, e.g., save does not reset "setModified" flag.

Reply:
It should do but i was trying to work out why resources that are not even loaded are marked as modified.. i.e. this can probably be removed but left in for now.